app/views/posts/_form.html.erb

```diff
@@ -154,7 +154,8 @@
     </div>
 
     <% if edit_comment %>
-      <div class="form-group">
+    <% if current_user.is_admin %>
```

Comment:
Probably better to use the [`admin?` helper](https://github.com/codidact/qpixel/blob/648a0fc85f494ab2a4700082b55a9a6c93275cc8/app/helpers/application_helper.rb#L9) here.